infrastructure/grpcutil/dial_options.go

```diff
@@ -0,0 +1,86 @@
+// Package grpcutil implements various utilities to simplify common gRPC APIs.
+package grpcutil
+
+import (
+	"context"
+	"crypto/tls"
+	"crypto/x509"
+	"fmt"
+
+	"github.com/certifi/gocertifi"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials"
```

Comment:
I know we had these anyway, indirectly, but are we adding any new direct dependencies?

Reply:
Feel free to resolve the conversation if that's no problem for you. :)